module/applications/activity/activity-usage-dialog.mjs

```diff
@@ -296,7 +296,7 @@ export default class ActivityUsageDialog extends Application5e {
     context.hasScaling = true;
     context.notes = [];
 
-    if ( this.activity.requiresSpellSlot ) {
+    if ( this.activity.requiresSpellSlot && (this.config.scaling !== false) ) {
```

Comment:
You should still be able to cast a spell with a higher level slot, even if the spell doesn't scale, no? Or perhaps I have misunderstood this section of the code.

Reply:
Spells are always able to scale, this just prevents scaling for specific use cases where they are activated from scrolls or tattoos.

Reply:
Ah ok, that sounds good then